roles/datadog-builder/tasks/datadog-builder.yml

```diff
@@ -20,12 +20,26 @@
     - /opt/venvs
     - /etc/datadog-builder
 
+- name: create datadog-builder group
+  group:
+    name: datadog-builder
+    state: present
+
 - name: create datadog-builder user
   user:
     name: datadog-builder
+    group: datadog-builder
```

Comment:
TIL ansible doesn't create a primary group with the same name as the user when creating a user.